app/src/main/kotlin/com/hippo/ehviewer/util/DateTimeUtil.kt

```diff
@@ -16,3 +17,16 @@ fun LocalDateTime.toEpochMillis(timeZone: TimeZone = TimeZone.UTC): Long =
 
 fun Long.toLocalDateTime(timeZone: TimeZone = TimeZone.UTC): LocalDateTime =
     Instant.fromEpochMilliseconds(this).toLocalDateTime(timeZone)
+
+// yyyy-MM-dd HH-mm
+val DATE_FORMAT_WITHOUT_SECONDS = LocalDateTime.Format {
+    year()
+    char('-')
+    monthNumber()
+    char('-')
+    dayOfMonth()
+    char(' ')
+    hour()
+    char('-')
+    minute()
+}
```

Comment:
https://github.com/FooIbar/EhViewer/blob/8d64a6f1b28c9152c0d982ea38311e30f5881704/app/src/main/kotlin/com/hippo/ehviewer/client/parser/ParserUtils.kt#L27 
Add a `parseDate` function in `ParserUtils` instead.